### PR TITLE
node_placement e2e test: make sure all virt-hanlers are working

### PR DESCRIPTION
**Pull request by nunnatsa**
The vm e2e test is failed very frequently. 

The node placement e2e test is changing the workloads settings in HCO, and restores them. This PR also makes sure that the virt handlers are running after the node placement test is done.

This maybe fix the failures in the vm test.

This PR is trying to fix it by making sure all the 
**What this PR does / why we need it**:

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket
https://issues.redhat.com/browse/CNV-46586
```

**Release note**:

```release-note
None
```

### CODE REVIEW

File: tests/func-tests/node_placement_test.go [hunk 1]
```diff
@@ -36,28 +36,30 @@ var _ = Describe("[rfe_id:4356][crit:medium][vendor:cnv-qe@redhat.com][level:sys
...
 	)
 
 	BeforeAll(func(ctx context.Context) {
 		cli = tests.GetControllerRuntimeClient()
 		cliSet = tests.GetK8sClientSet()
 
-		nodes := listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/control-plane!=")
-		tests.FailIfSingleNode(len(nodes.Items) < 2)
+		workerNodes = listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/worker")
+		//workerNodes = listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/control-plane!=")
```
Comment (orenc1) [inline tests/func-tests/node_placement_test.go:47]: is there a reason to keep this commented out?

(in a compact cluster, all 3 nodes have both `node-role.kubernetes.io/worker` and `node-role.kubernetes.io/control-plane` labels.)

File: tests/func-tests/node_placement_test.go [hunk 2]
```diff
@@ -36,28 +36,30 @@ var _ = Describe("[rfe_id:4356][crit:medium][vendor:cnv-qe@redhat.com][level:sys
...
+		tests.FailIfSingleNode(len(workerNodes.Items) < 2)
 
 		// Label all but first node with "node.kubernetes.io/hco-test-node-type=infra"
 		// We are doing this to remove dependency of this Describe block on a shell script that
 		// labels the nodes this way
 		Eventually(func(g Gomega, ctx context.Context) {
-			for _, node := range nodes.Items[:len(nodes.Items)-1] {
+			for _, node := range workerNodes.Items[:len(workerNodes.Items)-1] {
```
Comment (orenc1) [inline tests/func-tests/node_placement_test.go:54]: nit: this is labeling all but **last** node in the list, as opposed to the comment above.

File: tests/func-tests/node_placement_test.go [hunk 3]
```diff
@@ -36,28 +36,27 @@ var _ = Describe("[rfe_id:4356][crit:medium][vendor:cnv-qe@redhat.com][level:sys
...
 
-		nodes := listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/control-plane!=")
-		tests.FailIfSingleNode(len(nodes.Items) < 2)
+		workerNodes = listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/worker")
+		tests.FailIfSingleNode(len(workerNodes.Items) < 2)
 
-		// Label all but first node with "node.kubernetes.io/hco-test-node-type=infra"
-		// We are doing this to remove dependency of this Describe block on a shell script that
-		// labels the nodes this way
+		// Label all but the last node with "node.kubernetes.io/hco-test-node-type=infra"
```
Comment (orenc1) [inline tests/func-tests/node_placement_test.go:49]: now it's "all but the first node", no? 🙂

Comment (nunnatsa) [reply tests/func-tests/node_placement_test.go:49]: yes. sorry. fixed

Comment (nunnatsa) [reply tests/func-tests/node_placement_test.go:49]: It took me some time to understand that the error is in the comment and not in the code. leftover...

**Review by orenc1 — APPROVED** [review]
/approve

**Pull request merged**